samtranslator/model/sam_resources.py

```diff
@@ -290,6 +290,13 @@ def to_cloudformation(self, **kwargs):  # type: ignore[no-untyped-def]
                         self.logical_id,
                         "AutoPublishCodeSha256 must be a string",
                     )
+                # Lambda doesn't create a new version if the code in the unpublished version is the same as the
+                # previous published version. To handle scenario users only update content in 'CodeUri', CloudFormation
```

Comment:
nit: "To handle this scenario, users..."

Reply:
I don't think that's what I'm trying to convey. I want to say to handle the scenario that users only updated content in `CodeUri`, ..... Let me rephrase it.